dubbo-spring-boot/dubbo-spring-boot-compatible/actuator/src/main/resources/META-INF/spring/org.springframework.boot.autoconfigure.AutoConfiguration.imports

```diff
@@ -0,0 +1,3 @@
+org.apache.dubbo.spring.boot.actuate.autoconfigure.DubboEndpointAutoConfiguration
+org.apache.dubbo.spring.boot.actuate.autoconfigure.DubboHealthIndicatorAutoConfiguration
+org.apache.dubbo.spring.boot.actuate.autoconfigure.DubboEndpointMetadataAutoConfiguration
```

Comment:
I am more worried about compatibility issues in lower versions, such as version 2.7, which supports both AutoConfiguration.imports and spring.facts, whether it will be loaded twice.

Reply:
@CrazyHZM Don't worry, in Spring Boot 2.7.x, the `AutoConfiguration.imports` and `spring.factories` files will indeed be loaded at the same time, but they will all add to `Set<String> classNames`.

details see https://github.com/spring-projects/spring-boot/commit/08022ba86efa294f77574bda07851901b6c2b520